tests/Builders/ResumeThreeDSecureTwoBuilder.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Tests\Builders;
+
+class ResumeThreeDSecureTwoBuilder extends AbstractModelBuilder
+{
+    public function __construct($receiptId = '12345')
+    {
+        $this->attributeValues = array(
+            'receiptId'            => $receiptId
+        );
+    }
+
+    public function setCv2($cv2)
+    {
+        $this->setAttribute('cv2', $cv2);
+        return $this;
+    }
+
+    public function setThreeDSecureTwoFields($threeDSecure)
+    {
+        // Add any missing key
+        if (!array_key_exists('authenticationSource', $threeDSecure)) {
```

Comment:
It's only the methodCompletion attribute we want to send on in the resume request - the other attributes are only set on the initial payment/preauth request